cms/templates/cms/program_page.html

```diff
@@ -11,7 +12,8 @@
   <div class="app-media layout-boxed">
     <div class="main-content-wrapper">
       <section class="main-content">
-        <div class="banner-wrapper banner-sub banner-program-1">
+      {% image page.background_image fill-1310x613 as tmp_photo %}
```

Comment:
Yes, the user uploads the image, and we are using `wagtailimages_tags` to insert it into the template.